lisa/features/startstop.py

```diff
@@ -7,6 +7,12 @@
 FEATURE_NAME_STARTSTOP = "StartStop"
 
 
+class VMStatus(str, Enum):
+    Deallocated = "VM deallocated"
```

Comment:
The `Deallocated`, `Running`, `ProvisionSucceeded` are considered as common concepts, so they should be here. But the value like `VM deallocated`, `VM running`, `Provisioning succeeded` are Azure specific, so they should be a map in Azure feature.

Reply:
updated